spec/attr_vault/secret_spec.rb

```diff
@@ -30,5 +32,15 @@ def resolves_input(input)
     expect(
       secret.encryption_key
     ).to eq("B"*16)
+
+    secret = AttrVault::Secret.new(input, aead: true)
+    
+    expect {
+      secret.signing_key
+     }.to raise_error(AttrVault::Error)
```

Comment:
I should add linting so computers can tell you things like this and not me, but in the meantime I think you have an extra space before this brace.

Reply:
we could run the whole code base through standardrb. It has a --fix option.

Reply:
Sure, good idea. Though I'll do that separately.